### DISCUSSION TITLE
Daily 21/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#200] [General]
Ayer vi la clase de SGC pero no la de full-stack, voy a ver hoy el sincrónico. Armé una tarea de la cual se desprende cada tarea de SQL que hay que hacer con un query cada una, no tenemos queries de borrado y hay dos queries que son complicados a simple vista. También hice el video explicando lo de aws / fastapi / python, con todos los titubeos me quedó en hora y media. Hoy en lo posible voy a hacer el merge entre mi rama de fastapi y dev para que ya esté disponible para todos. También quiero seguir creando tareas de cosas que hay que hacer, por ejemplo una tarea para meter código en cada clase de python.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando para la evaluación de GP, también estuve bastante investigando sobre Wordpress y probando distintas herramientas y plantillas. Por la tarde-noche participé de las clases de SGC y de Full.
Hoy tuve clase sincrónica de Ingles. Estarè estudiando Python y repasando para la evaluación de GP. También miraré el video de la explicación de aws/fastapi/python.
